### PR TITLE
Update getName method in used in Language class

### PR DESCRIPTION
**Pull request by Mintype**
Hi! This pull request does everything that issue #1497 asks for, which is to clean up the getName() methods of many of the languages. Now they just will return the language name and not the parser name. Please tell me if I did anything wrong or if I have to change anything. Thanks a lot!

### CODE REVIEW

File: languages/csharp/src/main/java/de/jplag/csharp/CSharpLanguage.java [hunk 1]
```diff
@@ -25,7 +25,7 @@ public String[] suffixes() {
 
     @Override
     public String getName() {
-        return NAME;
+        return "C#";
```
Comment (tsaglam) [inline languages/csharp/src/main/java/de/jplag/csharp/CSharpLanguage.java:28]: With this change, you produce inconsistencies with the `name` constant. You should probably just adapt the constant value instead.

**Comment by tsaglam** [comment]
Just as general feedback, while granular committing is great, your commits are very atomic, meaning they encapsulate very few changes, which would make them generally considered a bit too fine-grained. This is not a problem for this PR though, as I can squash on merge.

**Comment by tsaglam** [comment]
>  Where do you think this PR will be included?

I see this as a v6.0.0 feature, we can probably just merge this after the v5.1.0 release.

**Comment by Mintype** [comment]
> You will also need to update all names here https://github.com/jplag/JPlag/blob/develop/report-viewer/src/model/Language.ts
> 
> Depending on whether this will be a 5.1.0 or a 6.0.0 feature, we should keep the old names, to not break highlighting in these languages. @tsaglam Where do you think this PR will be included?

Ok I'll do this!

**Comment by Mintype** [comment]
wait a second. i forked the main branch not develop so I cant find https://github.com/jplag/JPlag/blob/develop/report-viewer/src/model/Language.ts 

Should I close this pull request and make a new one with the right branch?

**Pull request closed without merging**